### DISCUSSION TITLE
可不可以在安装mod时自动安装或提示安装支持mod

### DISCUSSION THREAD

**Discussion by Harry-hq** [Meloong-Git/PCL#3179] [讨论]
可不可以在安装mod时自动安装支持mod，就是在安装mod是检查是否有支持mod，没有可不可以自动或提示下载

**Comment by allMagicNB** [comment]
没看懂你的意思，是想自动安装前置Mod吗？

**Reply by Harry-hq** [reply]
是的

**Comment by ghost** [comment]
安装支持是什么

**Comment by Ad-closeNN** [comment]
~~last month?!!~~
找到一个mod，如果有mod需要前置（比如iris shaders，需要sodium），就像自动安装mc一样勾选所需的前置mod并提示？

**Reply by Harry-hq** [reply]
是的

**Comment by Ad-closeNN** [comment]
你开一个新功能issue吧
但估计会变成投票.jpg


------ 原始信息 ------
来自 "Harry-hq" ***@***.***>
收件人 "Hex-Dragon/PCL2" ***@***.***>
抄送 "Ad_closeNN" ***@***.***>; "Comment" 
***@***.***>
日期 2024/2/22 19:10:46
主题 Re: [Hex-Dragon/PCL2] 可不可以在安装mod时自动安装或提示安装支持mod (Discussion #3179)

**Comment by allMagicNB** [comment]
#179
